yaml_files/simphony_metadata.yml

```diff
@@ -418,5 +418,36 @@ CUDS_KEYS:
     definition: Multiphase model
 
   ELECTROSTATIC_MODEL:
-    parent: CUBA.PHYSICS_EQUATION
     definition: Electrostatic model
+    parent: CUBA.PHYSICS_EQUATION
+    models: [CUBA.CFD, CUBA.MOLECULAR_DYNAMICS, CUBA.DEM]
+
+  BASIS:
+    definition: Space basis vectors
+    parent: CUBA.CUDS_COMPONENT
+    CUBA.VECTOR:
+      shape: (3, 3)
+      default: [[1, 0, 0], [0, 1, 0], [0, 0, 1]]
+
+  ORIGIN:
+    definition: The origin of a space system
+    parent: CUBA.CUDS_COMPONENT
+    CUBA.POINT:
+      default: [0, 0, 0]
+      scope: CUBA.USER
+
+  BOX:
```

Comment:
Box should have currently only the vector (of shape 3). Although we can define it as a simple cell, this should be avoided as the parent of BOX should be a Boundary! a cell is a low level computational object, while the metadata should reflect the logic of the scientific end user problem.